api/src/main/java/org/openmrs/api/ProgramWorkflowService.java

```diff
@@ -530,4 +535,36 @@ public void purgeConceptStateConversion(ConceptStateConversion conceptStateConve
 	 * @should throw an error when multiple program states with same uuid are found
 	 */
 	public ConceptStateConversion getConceptStateConversionByUuid(String uuid);
+
+        @Transactional(readOnly = true)
+        @Authorized({"View Program Attribute Types"})
```

Comment:
For privileges, can you change "View" to "Get"?

Reply:
Okay